contents.json

```diff
@@ -2680,6 +2680,16 @@
 		"description": "Answers to common \"How do I test this with UI Testing?\" questions with a working example app.",
 		"homepage": "https://github.com/joemasilotti/UI-Testing-Cheat-Sheet"
 	}, {
+		"title": "Apodimark",
+		"category": "text",
+		"description": "Fast, flexible markdown parser written in Swift",
+		"homepage": "https://github.com/loiclec/Apodimark"
+	}, {
+		"title": "CocoaMarkdown",
```

Comment:
@dkhamsing noted. I have removed it and kept the Apodimark entry. Please advice.